docs/examples/custom_operations/numba_function.ipynb

```diff
@@ -0,0 +1,468 @@
+{
```

Comment:
<p> pipeline which <strong>rotates</strong> images by 90 degrees</p><p> </p><p>Since the output shape will be different than the input <strong>shape</strong></p><p> </p><p>In the setup function, we <strong>define </strong>the output</p><p> </p><p>shapes of the outputs are the same as <s>the</s> in the inputs. </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA/DALI/pull/2886/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>